README.yaml

```diff
@@ -55,33 +69,55 @@ references:
     description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration"
     url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
 
-# Short description of this project
-description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
-  use when creating new repositories.
 
 # Introduction to the project
 #introduction: |-
 #  This is an introduction.
 
 # How to use this module. Should be an easy example to copy and paste.
-usage: |-
+uusage: |-
   For a complete example, see [examples/complete](examples/complete).
-  
   For automated tests of the complete example using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
   (which tests and deploys the example on AWS), see [test](test).
-
   ```hcl
-  module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
-    example = "Hello world!"
-  }
-  ```
-
+      module "registration" {
+          source = "./registration"
+          enabled = true
+          name    = "clienta-org"
+          namespace = ""
```

Comment:
Each client should have it's own kubernetes namespace. You can use the client name.
```
          name    = "clienta-org"
          namespace = "clienta-org"
```